optimum/exporters/openvino/__main__.py

```diff
@@ -299,5 +311,18 @@ def main_export(
         output_names=files_subpaths,
         input_shapes=input_shapes,
         device=device,
+        fp16=fp16,
         model_kwargs=model_kwargs,
     )
+    del models_and_onnx_configs
+
+    if int8:
```

Comment:
One comment from my side. Such an implementation means that we will:
- Convert from PyTorch to OpenVINO with memory reuse (no-copy)
- However, after that, we will serialize to the disk, load the model again, quantize weights, remove files from the disk, and store the compressed version, finally.
- It can be time-consuming for really large models.

Ideally, we should have nncf.compress_weights() right after openvino.convert_model(), the similar to FP16.

Reply:
I opened a follow-up PR #444 for the default 8-bit compression. I tried to follow the approach described above.

Reply:
@eaidova, please take a look as well.

Reply:
> Ideally, we should have nncf.compress_weights() right after openvino.convert_model(), the similar to FP16.

Yes that's good point, was going to modify this PR but I see that you opened https://github.com/huggingface/optimum-intel/pull/444, thanks @AlexKoff88